modules/social_features/social_event/modules/social_event_an_enroll/social_event_an_enroll.module

```diff
@@ -201,10 +202,10 @@ function social_event_an_enroll_menu_local_tasks_alter(&$data, $route_name) {
 function social_event_an_enroll_is_enabled(Node $node) {
   $config = \Drupal::config('social_event_an_enroll.settings');
   $is_global_enabled = $config->get('event_an_enroll');
-  $is_event = $node->getType() === 'event';
+  $is_event = $node instanceof Event;
   $is_public = $node->get('field_content_visibility')->getString() === 'public';
   $is_event_an_enroll = !empty($node->get('field_event_an_enroll')->value);
-  $is_event_enroll = \Drupal::service('social_event.enroll')->isEnabled($node);
+  $is_event_enroll = $is_event && $node->isEnrollmentEnabled();
```

Comment:
@nechai Could you explain why this change was needed?

Reply:
This is refactoring change. As I added bundle class to event we can check if event have enrolments enabled inside an event object. Also, in my opinion  the method name `isEnrollmentEnabled()` has more sense rather then `isEnabled()` 🙂